packaging/rhel/syslog-ng.spec

```diff
@@ -418,8 +418,8 @@ fi
 %{_datadir}/%{name}/tools/
 
 %changelog
-* Tue Feb 26 2019 Gabor Nagy <gabor.nagy@balabit.com> - 3.20.1-1
-- update to 3.20.1
+* Tue Feb 26 2019 Gabor Nagy <gabor.nagy@balabit.com> - 3.21.1-1
+- update to 3.21.1
```

Comment:
The date does not match with the version, I'd append the changelog with a new record.

Reply:
updated

Reply:
Thanks